templates/CRM/Contribute/Form/Search/Common.tpl

```diff
@@ -160,11 +160,6 @@
 {include file="CRM/Campaign/Form/addCampaignToComponent.tpl" campaignContext="componentSearch"
 campaignTrClass='' campaignTdClass=''}
 
-<tr>
-  <td colspan="2">
-{include file="CRM/Contribute/Form/Search/ContributionRecur.tpl"}
```

Comment:
Removing this include seems wrong. I'm not seeing the recurring contribution section at all in Find Contributions and Advanced Search.